int32 fiducial_id
float64 x
float64 y
float64 z